internal/controller/compute/user/user_test.go

```diff
@@ -0,0 +1,156 @@
+package user
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"testing"
+
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	"github.com/crossplane/crossplane-runtime/pkg/logging"
+	"github.com/crossplane/crossplane-runtime/pkg/meta"
+	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+	"github.com/golang/mock/gomock"
+	"github.com/ionos-cloud/crossplane-provider-ionoscloud/apis/compute/v1alpha1"
+	usermock "github.com/ionos-cloud/crossplane-provider-ionoscloud/internal/mock/clients/compute/user"
+	ionoscloud "github.com/ionos-cloud/sdk-go/v6"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+func TestUserObserve(t *testing.T) {
```

Comment:
No tests for Create, Update and Delete.

Reply:
I said that in the PR description. WIP. tests are coming.